### ISSUE TITLE
TextEditor.get is deprecated.

### ISSUE THREAD

**Issue by BlaM** [klorenz/atom-regex-railroad-diagrams#80]
I get this message from Atom's [Deprecation Cop](https://atom.io/packages/deprecation-cop):

`TextEditor.prototype.displayBuffer` has always been private, but now
it is gone. Reading the `displayBuffer` property now returns a reference
to the containing `TextEditor`, which now provides *some* of the API of
the defunct `DisplayBuffer` class.
```
TextEditor.get (C:\Users\BlaM\AppData\Local\atom\app-1.12.5\resources\app.asar\src\text-editor.js:100:14)
Object.bufferRangeForScope (C:\Users\BlaM\.atom\packages\regex-railroad-diagram\lib\regex-railroad-diagram.coffee:94:22)
Object.getRegexpBufferRange (C:\Users\BlaM\.atom\packages\regex-railroad-diagram\lib\regex-railroad-diagram.coffee:111:20)
Object.checkForRegExp (C:\Users\BlaM\.atom\packages\regex-railroad-diagram\lib\regex-railroad-diagram.coffee:164:20)
<unknown> (C:\Users\BlaM\.atom\packages\regex-railroad-diagram\lib\regex-railroad-diagram.coffee:34:26)
later (C:\Users\BlaM\AppData\Local\atom\app-1.12.5\resources\app.asar\node_modules\underscore\underscore.js:724:25)
```

**Comment by igs-awilliams** [comment]
+1

**Comment by foisonocean** [comment]
Please fix it.

**Comment by mistadikay** [comment]
There is a PR for this already: #83 If the author of this plugin is not willing to support it anymore, he might consider to give someone else access to it. ping @klorenz 🙂

**Comment by jediorange** [comment]
@klorenz seems to have disappeared, no activity since October. I hope he's OK... ☹️

**Comment by nighter233** [comment]
+1

**Comment by quanru** [comment]
+1

**Comment by SavageCore** [comment]
There is a maintaned fork now: https://github.com/mikesprague/atom-regex-railroad-diagrams-fork

**Comment by sunlery** [comment]
+1

**Comment by mikesprague** [comment]
Closed by pull request #88
